tests/cypress/integration/actions_tasks_objects/case_40_create_task_without_necessary_arguments.js

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Try to create a task without necessary arguments.', () => {
+    const caseId = '40';
+    const labelName = `Case ${caseId}`;
+    const taskName = `New annotation task for ${labelName}`;
+    const imagesCount = 1;
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+
+    before(() => {
+        cy.visit('auth/login');
+        cy.login();
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+        cy.createZipArchive(directoryToArchive, archivePath);
+    });
+
+    after(() => {
+        cy.goToTaskList();
+        cy.getTaskID(taskName).then(($taskID) => {
+            cy.deleteTask(taskName, $taskID);
+        });
+    });
+
+    describe(`Testing "${labelName}"`, () => {
+        it('Go to create task page.', () => {
+            cy.get('#cvat-create-task-button').click();
```

Comment:
Should be a part of before

Reply:
Click to "Create new task" button moved to before.